docs/maya/simple_setup.md

```diff
@@ -63,6 +63,50 @@ The order of painting is important because after every stroke a normalisation of
 
+## AdnFat
+
+To create a basic scenario using the AdnFat deformer, start with a scene with the following elements:
+
+  - One base mesh to which the fat layer will be attached to. This could be for example the fascia.
+  - A fat tissue mesh without animation or deformation.
+
+The AdnFat deformer will get applied to the second selected mesh which will become the simulated mesh (the fat tissue).
+
+<figure>
+  <img src="images/setup_fat_0.png"> 
+  <figcaption><b>Figure 7</b>: Basic setup for fat simulations.</figcaption>
+</figure>
+
+> [!NOTE]
+> - The base mesh and the fat mesh must have the same vertex count and triangulation.
+> - The base mesh must be provided, otherwise the Fat solver will abort the simulation.
+
+### Create Deformer
+
+To create the AdnFat deformer first select the base mesh and then the fat tissue mesh. Then press the ![AdnFat](images/adn_fat.png){style="width:4%"} shelf button or go to AdonisFX Menu > *Fat*.
+
+To create the AdnFat deformer with some initial customisation, double-click the shelf button or press the option box in the menu item. This will display a pop-up window that will allow to do some initial customisation, as well as creating the deformer with a custom name. Once all data has been provided press the *Create* button and the deformer will get created.
+
+<figure>
+  <img src="images/setup_fat_1.png"> 
+  <figcaption><b>Figure 8</b>: AdnFat deformer creation scenario.</figcaption>
+</figure>
+
+After basic configuration, to alter the dynamics of the fat layer (adding or reducing the jiggle effect) it is advisable to tweak the main attributes like; *Iterations*, *Substeps*, *Global Damping Multiplier* or the per-constraints stiffness values in the *Override Constraint stiffness* section.
+
+### Paint Weights
+
+With the default paint setup provided when creating a new AdnFat, the simulation should already create plausible results. However, below we walk you through the 2 main maps that can be altered to modify the behavior of the fat simulation.
+
+In the case of the AdnFat deformer use the Maya Paint tool to setup and paint its paintable weight attributes. The most important paintable maps are *Shape Preservation* and *Volume Shape Preservation* both with a flooded value of 1.0 by default.
+With the *Shape Preservation* being flooded to 1.0 by default the solver will try to maintain the internal structural shape properties of the fat layer. Reducing this map's values can increase the amount of jiggling in the fat tissue in combination with different other parameters. However, reducing the *Shape Preservation* map can decrease the fat layer's ability to maintain its shape. Finding the right balance will allow you to get the desired results. It may be advisable to keep this map flooded to 1.0 and reduce its value in areas that don't require any structural shape preservation.
```

Comment:
Woud you split all of this into two parapgraphs? One for each map?

Reply:
Agreed! Done!